app/components/App.js

```diff
@@ -5,11 +5,15 @@ const Router = ReactRouter.BrowserRouter;
 const Route = ReactRouter.Route;
 const Switch = ReactRouter.Switch;
 const Game = require('./Game');
+const pathFind = require('../utils/BFS');
```

Comment:
so I've been googling on the difference between `require()` and `import` and got into a wall of not understanding the difference between 'dynamic' and 'static' loading. And that's besides the fact that most places say that `import` is being transpiled by Babel to `require()` anyway because there is no native support yet... :/

1. Any chance for a link to clarify that difference between dynamic and static loading?

2. you marked only the `findPath` to use with `import`. Is there any reason not to use `import` everywhere instead of the `require()` I've been using to for the other classes of the app? Or `React` for that matter?

Reply:
1. It's a complex topic and I can write you the basics in chat.
2. I only commented in this PR on changes, not on everything. If I'd comment on everything, we'd never finish. Gotta limit the scope, otherwise it's unmanageable. You should use `import` everywhere you can.